doc/userguide/src/ExecutingTestCases/OutputFiles.rst

```diff
@@ -552,6 +552,26 @@ deeply nested keyword structures.
 
 .. note:: `TAG:<pattern>` mode was added in Robot Framework 2.9.
 
+Automatically expanded keywords in log.html
+-------------------------------------------
+
+The :option:`--autoexpandkeywords` option set keywords initially to visible in log.html.
+Autoexpandkeywords supports the following modes:
+
+`NAME:<pattern>`
+   Expand keywords matching the given pattern. Pattern matching rules are
+   same as when `removing keywords`_ using `NAME:<pattern>` mode.
+
+`TAG:<pattern>`
+   Expand keywords with tags matching the given pattern. Pattern matching
+   rules are same as when `removing keywords`_ using `TAG:<pattern>` mode.
+
+Examples::
+
+   rebot --autoexpandkeywords name:CapturePageScreenshot output.xml
+   rebot --autoexpandkeywords tag:ShowInLog output.xml
+
```

Comment:
Need to add a note somewhere that this is a new feature in RF 3.1. There ought to be other features that have similar notes that can be used as examples.

Reply:
added a note to be new in Robot Framework 3.2

Reply:
updated with note new in RF 3.2